polaris/curation/_data_curator.py

```diff
@@ -32,10 +36,11 @@ def _identify_stereoisomers_with_activity_cliff(
         mol_hash_ids_with_cliff: A list mol hash ids for the stereoisomer molecules which show activity cliff.
 
     """
-    data_type = type_of_target(data[data_col].dropna().values, input_name=data_col)
 
     if data_type is None:
         raise ValueError(f"The column {data_col} contains less than 2 unique values.")
+    if CLASS_PREFIX in data_col:
```

Comment:
For such global variables, I think we should add a Pydantic Settings class. Similar to [how we did this for the client](https://github.com/polaris-hub/polaris/blob/main/polaris/hub/settings.py). That way, a user could configure them through e.g. environment variables!